st2common/st2common/util/config_loader.py

```diff
@@ -35,6 +35,8 @@
 
 LOG = logging.getLogger(__name__)
 
+DEFAULT_VALUE_MAGIC_MARKER = '~~***---%%--%%--%--%%%***~~~+++'
```

Comment:
I'm starting getting it :spades: 
Also a small `#` code comment would be useful to clarify.